legend-sdlc-test-generation/src/test/java/org/finos/legend/sdlc/test/junit/TestJUnitTestGeneratorSerialization.java

```diff
@@ -61,19 +61,19 @@ public static void cleanUp() throws Exception
     @Test
     public void testWriteTestClassesWithoutRootPackage() throws IOException
     {
-        testWriteTestClasses(null, "generated/java/execution/TestRelationalMapping.java", "generated/java/legend/demo/TestSingleQuoteInResultM2M.java", "generated/java/model/mapping/TestSourceToTargetM2M.java", "generated/java/testTestSuites/TestTestService.java", "generated/java/testTestSuites/TestTestService2.java", "generated/java/testTestSuites/TestServiceStoreMapping.java");
+        testWriteTestClasses(null, "generated/java/execution/TestRelationalMapping.java", "generated/java/execution/TestRelationalMapping.java", "generated/java/legend/demo/TestSingleQuoteInResultM2M.java", "generated/java/model/mapping/TestSourceToTargetM2M.java", "generated/java/testTestSuites/TestTestService.java", "generated/java/testTestSuites/TestTestService2.java", "generated/java/testTestSuites/TestServiceStoreMapping.java", "generated/java/testTestSuites/TestMyServiceIsVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryV953300342.java");
```

Comment:
You also need tests for names with unicode characters that require multiple bytes to encode, including one that spans the byte limit.